### PR TITLE
[script.reddit.reader] 0.5.7

### PR DESCRIPTION
**Pull request by gedisony**

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
@phil65 @learningit (or any one else) some comments?

File: script.reddit.reader/addon.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    id="script.reddit.reader"
+    name="Reddit Reader"
+    version="0.5.7"
+    provider-name="gedisony">
+    <requires>
+        <import addon="xbmc.python"                     version="2.14.0"                />
+        <import addon="xbmc.addon"                      version="13.0.0"                />
+        <import addon="plugin.video.youtube"            version="5.0.0"                 />
+        <import addon="plugin.video.vimeo"              version="4.0.0" optional="true" />
+        <import addon="plugin.video.dailymotion_com"    version="1.0.0" optional="true" />
```
Comment (phil65) [inline script.reddit.reader/addon.xml]: Why do you require all these addons? Shouldnt script.module.youtube.dl be able to deal with most of it?

File: script.reddit.reader/addon.xml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        <import addon="script.module.simple.downloader" version="1.0.0" />
+        <import addon="plugin.program.chrome.launcher"  version="1.0.0" optional="true" />
+	    <import addon="script.module.requests"          version="1.1.0"/>
+	    <import addon="script.module.youtube.dl"        version="16.521.0"/>
+	    <import addon="script.module.beautifulsoup"     version="3.2.1"/>
+    </requires>
+
+	<extension point="xbmc.python.script" library="default.py">
```
Comment (phil65) [inline script.reddit.reader/addon.xml]: only executable, please.

File: script.reddit.reader/default.py [hunk 3]
```diff
@@ -0,0 +1,2253 @@
...
+#     li.sort( key=getKey )
+#     log("  sorted")
+#     
+#     for l in li:
+#         log( '   %s-%s'  %(l.getLabel(), l.getProperty('onClick_action') ) )
+            
+        
+    ui = commentsGUI('view_461_comments.xml' , addon_path, defaultSkin='Default', defaultRes='1080i', listing=li, id=55)
+    #NOTE: the subreddit selection screen and comments screen use the same gui. there is a button that is only for the comments screen
+    ui.setProperty('comments', 'yes')   #i cannot get the links button to show/hide in the gui class. I resort to setting a proper
```
Comment (phil65) [inline script.reddit.reader/default.py]: directly connecting to our databases is not allowed.

File: script.reddit.reader/addon.xml (hunk 1 above)
Comment (gedisony) [inline script.reddit.reader/addon.xml]: The wait time to start playing the video is shorter using the video plugins.

File: script.reddit.reader/addon.xml (hunk 2 above)
Comment (gedisony) [inline script.reddit.reader/addon.xml]: I will remove the video and image "<provides>"

File: script.reddit.reader/addon.xml (hunk 1 above)
Comment (phil65) [inline script.reddit.reader/addon.xml]: please only use script.module.youtube.dl  if possible.

**Comment by gedisony** [comment]
I've done the changes. Do I just close this pull request and do another one?

**Comment by phil65** [comment]
you can also update this pull-request by force-pushing, if you hknow how to do that.

**Pull request closed without merging**